lib/diamond/utils.ts

```diff
@@ -22,13 +22,15 @@ export interface FacetCut extends Facet {
 }
 
 // returns a list of signatures for a contract
-export function getSignatures(contract: Contract): string[] {
+export function getFuntionSignatures(contract: Contract): string[] {
```

Comment:
Spelling

Reply:
this was fixed in https://github.com/solidstate-network/solidstate-solidity/pull/158/commits/dbf11d58067b3f5b081ef0d26414905f825fc4c1.

Reply:
Clicking the notification only showed this commit.  I feel like it used to show all new commits.

Reply:
Maybe I had the tab open since before you fixed?

Reply:
Probably because I edited the comment after fixing it

Reply:
ha, okay